http_stubs/models.py

```diff
@@ -85,6 +85,13 @@ class HTTPStub(models.Model):
         default=dict,
         blank=True,
     )
+    request_script = models.TextField(
+        verbose_name='BETA: Request script',
+        help_text='DANGEROUS ZONE: This feature is unstable and can completely'
+                  ' destroy this service. Language - python 3.8. The script'
+                  ' will run on every call.',
```

Comment:
Unsure (it's a tricky part of English grammar) but it sounds more correct.
```suggestion
                  ' will be run on every call.',
```